src/plugin.js

```diff
@@ -497,7 +498,9 @@ class VR extends Plugin {
             canvas: this.renderedCanvas,
             orientation: videojs.browser.IS_IOS || videojs.browser.IS_ANDROID || false
           });
+          this.canvasPlayerControls = new CanvasPlayerControls(this.player_, this.renderedCanvas);
```

Comment:
should we add this as a child of the player rather than just instantiating it here?

Reply:
oh, it's not a real component, just an object.